### DISCUSSION TITLE
Way to configure server auto-restart time?

### DISCUSSION THREAD

**Discussion by altrae** [lloesche/valheim-server-docker#187] [Q&A]
My server restarts every day even while players are connected at 5am UTC which is, unfortunately, 11pm my time. Is there a way to configure this? I've looked at the logs and think the server may be performing an update though I'm not very familiar with Docker or these scripts. I also tried looking at the server scripts themselves but haven't been able to find what exactly is triggering the restart because it looked to me as if it should be checking that no one is on before performing an update. If this is true, then there may be a bug because it has happened with one or more players connected consistently.

Example from attached log:
**2021-02-28 05:00:00,stdout,Feb 28 05:00:00 lloesche-valheim-server1 cron.info crond[10]: USER root pid 28896 cmd /usr/bin/supervisorctl restart valheim-server**

[lloesche-valheim-server1.zip](https://github.com/lloesche/valheim-server-docker/files/6097669/lloesche-valheim-server1.zip)

**Comment by lloesche** [comment]
You likely just want to set `TZ` to whatever your timezone is so that 5 AM is actually 5 AM.
See https://github.com/lloesche/valheim-server-docker#environment-variables
Alternatively or in addition check `RESTART_CRON`.

**Reply by altrae** [reply]
Awesome, thank you! I will try this and update the result here.

**Reply by adaMATTium** [reply]
I updated the TZ on mine to America/Chicago, and it appears it now executes the cronjob at 11 AM. So it does seem that you need to modify the cronjob time as well.

Edit: Nevermind. I am dumb. Docker logs are in UTC, I just realized.
